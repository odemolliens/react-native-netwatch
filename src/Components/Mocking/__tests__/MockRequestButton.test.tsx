import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { MockRequestButton } from '../MockRequestButton';

describe('Test MockRequestButton', () => {
  let component: ShallowWrapper;

  it('should render properly', () => {
    component = shallow(<MockRequestButton onPress={jest.fn()} />);
    expect(component).toMatchSnapshot();
  });
});
