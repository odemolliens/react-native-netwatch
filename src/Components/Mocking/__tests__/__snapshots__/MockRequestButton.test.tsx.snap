// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test MockRequestButton should render properly 1`] = `
<ForwardRef
  onPress={[MockFunction]}
  style={
    Object {
      "alignItems": "center",
      "borderLeftWidth": 0,
      "flexDirection": "row",
    }
  }
  testID="buttonMockRequest"
>
  <Icon
    allowFontScaling={false}
    color="#06B6D4"
    name="clipboard-arrow-left-outline"
    size={14}
  />
  <Text
    style={
      Array [
        Object {
          "color": "#06B6D4",
          "marginLeft": 6,
        },
      ]
    }
  >
    Mock Request
  </Text>
</ForwardRef>
`;
